aws/rust-runtime/aws-inlineable/src/presigning.rs

```diff
@@ -262,12 +262,6 @@ pub(crate) mod service {
         fn call(&mut self, req: operation::Request) -> Self::Future {
             let (mut req, _) = req.into_parts();
 
-            // Remove headers from input serialization that shouldn't be part of the presigned
-            // request since the request body is unsigned and left up to the person making the final
-            // HTTP request.
-            req.headers_mut().remove(CONTENT_LENGTH);
-            req.headers_mut().remove(CONTENT_TYPE);
-
             // Remove user agent headers since the request will not be executed by the AWS Rust SDK.
             req.headers_mut().remove(USER_AGENT);
             req.headers_mut().remove("X-Amz-User-Agent");
```

Comment:
maybe this is hard...this is probably hard...ideally we just wouldn't hit the UA middleware

Reply:
We could potentially add something to the property bag that tells the UA middleware to no-op, but I don't think it would be easy to customize the middleware in the presigning function as it currently is.